test/metabase/lib/expression_test.cljc

```diff
@@ -545,7 +544,7 @@
 (deftest ^:parallel diagnose-expression-literals-test
   (testing "top-level literals are not allowed"
     (let [query (lib/query meta/metadata-provider (meta/table-metadata :orders))
-          expr  [:value {:lib/uuid (str (random-uuid)) :effective-type :type/Integer} 1]]
+          expr  (lib.expression/value true)]
```

Comment:
Previously I tried to use a raw integer for a filter, which should have been rejected. Now, with type checking, it's an invalid value. I used a boolean literal which should be supported by both custom column and filter expressions.